.github/workflows/release-chart.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```

Comment:
This doesn't feel it belongs to this commit. Maybe a general (separate) commit to bump up the version of this action?

Reply:
I've removed it for now. Can do in a different MR.